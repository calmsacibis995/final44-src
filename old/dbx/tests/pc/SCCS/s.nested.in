h18543
s 00011/00000/00000
d D 5.1 90/06/25 15:37:13 bostic 1 0
c date and time created 90/06/25 15:37:13 by bostic
e
u
U
t
T
I 1
whereis i
whereis j
stop at 18
run
which k
print i, j, nested.i, p.i, p.j
step
step
print i, j, nested.i
cont
print nested.i
E 1
