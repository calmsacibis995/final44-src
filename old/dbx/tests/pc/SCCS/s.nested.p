h33403
s 00030/00000/00000
d D 5.1 90/06/25 15:37:16 bostic 1 0
c date and time created 90/06/25 15:37:16 by bostic
e
u
U
t
T
I 1
(*
 * Test of nested procedures and modules.
 *)

program nested (input, output);
var
    i, k : integer;

procedure p (var i : integer);
var
    j, k : integer;

    procedure nestedp (var j : integer);
    var
	i : integer;
    begin
	i := j + 2;
	j := i;
    end;

begin
    j := i + 1;
    nestedp(j);
    i := j;
end;

begin
    i := 3;
    p(i);
end.
E 1
