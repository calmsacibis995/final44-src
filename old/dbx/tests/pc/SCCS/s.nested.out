h28721
s 00017/00000/00000
d D 5.1 90/06/25 15:37:15 bostic 1 0
c date and time created 90/06/25 15:37:15 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
p.i p.nestedp.i nested.i
p.j p.nestedp.j
[1] stop at 18
[1] stopped in nestedp at line 18
   18   	j := i;
p.k
6 4 3 3 4 
stopped in nestedp at line 19
   19       end;
stopped in p at line 24
   24       i := j;
3 6 3 

execution completed
6 

E 1
